core/src/main/java/org/elasticsearch/cluster/action/index/NodeMappingRefreshAction.java

```diff
@@ -85,7 +83,6 @@ public NodeMappingRefreshRequest() {
         public NodeMappingRefreshRequest(String index, String indexUUID, String[] types, String nodeId) {
             this.index = index;
             this.indexUUID = indexUUID;
-            this.types = types;
```

Comment:
Care will need to be taken here when backporting to 2.x lest backwards compatibility be broken. :)